beacon-chain/p2p/subnets.go

```diff
@@ -15,8 +17,17 @@ import (
 )
 
 var attestationSubnetCount = params.BeaconNetworkConfig().AttestationSubnetCount
+var syncCommsSubnetCount = params.BeaconConfig().SyncCommitteeSubnetCount
```

Comment:
This assumes that the subnet count doesn't change. Currently there is no difference between the subnet counts for both mainnet/minimal.

Reply:
Right, if we one `AttestationSubnetCount` or `SyncCommitteeSubnetCount` change between configs (ie mainnet, minimal, testnet config), it'll be hard to discover this is an issue right?

We can keep it, but we should display failure if that's the case rather than silence

Reply:
Alright so I reinitialize them again in our service constructor, let me know if that works for you.

Reply:
Much better : )